docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -74,6 +74,17 @@ relative to when data was written last to a rolled over index.
 The previous phase's actions must complete before {ILM} will check `min_age` and
 transition into the next phase.
 
+=== Phase Execution
+
+The current phase definition, of an index's policy being executed, is stored
+in the index's metadata. The phase and its actions are compiled into a series
+of discrete steps that are executed. These steps can be thought of as distinct
+API calls to Elasticsearch. Since some {ILM} actions are more complex and involve
```

Comment:
I don't think we should encourage users to think of each step as an API call as its not true and could leads to confusion. I wonder if we can word this differently?

Reply:
I was iffy about it too, the sentence after mentions how actions are complex and need to be broken down, so maybe the sentence can simply be deleted

Reply:
deleted it